arrocke/gloss-translation
# feat: suggestions from google translate

## What has changed

* generates machine translations using Google Translate with the English glosses
* machine translations are cached in the database
* machine suggestions are marked in the user interface with the Google icon



## Connected Issues

closes #210 



## QA Steps

1. Navigate to the translation view
2. Observe glosses marked with the Google icon
3. Approve a gloss
4. Observe approved gloss and Google suggestion in dropdown



## Post-Deployment
